auth/service.cc

```diff
@@ -76,6 +78,12 @@ class auth_migration_listener final : public ::service::migration_listener {
         }).handle_exception([] (std::exception_ptr e) {
             log.error("Unexpected exception while revoking all permissions on dropped keyspace: {}", e);
         });
+        (void)_authorizer.revoke_all(
+            auth::make_functions_resource(ks_name)).handle_exception_type([](const unsupported_authorization_operation&) {
+            // Nothing.
+        }).handle_exception([] (std::exception_ptr e) {
+            log.error("Unexpected exception while revoking all permissions on functions in dropped keyspace: {}", e);
```

Comment:
It would be nice to include the keyspace name too, but I see other error messages don't include it either. Can be changed all at once in a follow-up.